main/src/ca/uwaterloo/flix/language/ast/UnkindedType.scala

```diff
@@ -359,8 +359,8 @@ object UnkindedType {
   /**
     * Constructs the type a -> b \ IO
     */
-  def mkImpureArrow(a: UnkindedType, b: UnkindedType, loc: SourceLocation): UnkindedType = {
-    val eff = Some(UnkindedType.Cst(TypeConstructor.EffUniv, loc))
+  def mkIoArrow(a: UnkindedType, b: UnkindedType, loc: SourceLocation): UnkindedType = {
+    val eff = Some(UnkindedType.Cst(TypeConstructor.Effect(new Symbol.EffectSym(Nil, "IO", SourceLocation.Unknown)), loc))
```

Comment:
Perhaps we should move the hardcoded IO symbol to the Symbol.scala class? I think that would be good.